pkgs/data/fonts/redhat-liberation-fonts/default.nix

```diff
@@ -50,36 +50,36 @@ let
 
 in {
   liberation_ttf_v1_from_source = common rec {
-    version = "1.07.4";
-    url = "https://releases.pagure.org/liberation-fonts/liberation-fonts-${version}.tar.gz";
-    sha256 = "01jlg88q2s6by7qv6fmnrlx0lwjarrjrpxv811zjz6f2im4vg65d";
+    version = "1.07.5";
+    url = https://github.com/liberationfonts/liberation-1.7-fonts/files/2175696/liberation-fonts-1.07.5.tar.gz;
```

Comment:
maybe ```fetchFromGitHub { ...; rev = version; }``` for source versions ?
Repos have release tags (https://github.com/liberationfonts/liberation-1.7-fonts/releases https://github.com/liberationfonts/liberation-fonts/releases)